adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/interpreter/CsvInterpreter.java

```diff
@@ -57,12 +57,12 @@ protected void validateParameters(Map<String, Object> inputParameters) {
   }
 
   @Override
-  protected JsonNode doInterpret(String data, Map<String, Object> parameters) throws IOException {
+  protected String doInterpret(String data, Map<String, Object> parameters) throws IOException {
```

Comment:
Can we undo this change since we have no raw interpreter anymore?

Reply:
Yeah, good point, I will do that. The JsonNode return type was more precise.